src/Pages/ProductListing/ProductListing.css

```diff
@@ -0,0 +1,96 @@
+@import url('/src/index.css');
+
+/********** filter ********/
+
+.filters {
+    float: left;
+    height: 88.5vh;
+    width: 17vw;
+    position: fixed;
+    top: 8.5vh;
```

Comment:
Why use vh in top left or right properties ? I guess u should use % or rem
It will also help in responsiveness

Reply:
vh and vw did help in responsiveness but I guess % would be a better choice. Thank you for the suggestions. I will keep that in mind and apply them.